src/sss_client/pam_sss.c

```diff
@@ -995,7 +995,7 @@ static int eval_response(pam_handle_t *pamh, size_t buflen, uint8_t *buf,
                     break;
                 }
 
-                if (type == SSS_PAM_CERT_INFO && pi->cert_user == '\0') {
+                if (type == SSS_PAM_CERT_INFO && *pi->cert_user == '\0') {
```

Comment:
The reason I asked the change is because I'm quite in favour of self-contained changes that follow the coding style already present in the file.

For instance, take a look at line [1009](https://github.com/SSSD/sssd/pull/298/commits/c39f97a49a3adee7c318846a9bafe2e8a1456f4f#diff-3800d3e0355e2a056cc0210c1b298bc0L1009) for instance.

However, taking a look in the file more carefully, seems that line 1009 is the only part using *(foo->bar) == '\0' ... so, I'm taking the comment back ...